### PR TITLE
Feat/boosted pools

### PR DESCRIPTION
**Pull request by franzns**

### CODE REVIEW

File: modules/sources/contracts/fetch-erc4626-token-data.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            symbol: string;
+            isErc4626: boolean;
+            underlyingAddress?: string;
+        };
+    } = {};
+
+    for (const token of tokens) {
+        let response;
+        try {
+            response = await viemClient.multicall({
```
Comment (gmbronco) [inline modules/sources/contracts/fetch-erc4626-token-data.ts]: that will generate a lot of RPC calls when fetching more tokens. batch with multicaller?

File: modules/sources/contracts/fetch-erc4626-token-data.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                contracts: [
+                    {
+                        address: token.address as `0x${string}`,
+                        abi: MinimalErc4626Abi,
+                        functionName: 'asset',
+                    },
```
Comment (gmbronco) [inline modules/sources/contracts/fetch-erc4626-token-data.ts]: it's just the first call is used, are the others to get the rate?

Comment (franzns) [reply modules/sources/contracts/fetch-erc4626-token-data.ts]: the others are just to determine that it is an ERC4626 token. These are the same things that the vault also checks

Comment (gmbronco) [reply modules/sources/contracts/fetch-erc4626-token-data.ts]: ah, it's checking if the whole interface is implemented? try, catch, continue - i see

**Pull request merged**